### ISSUE TITLE
[question] Emulating mouse and Xbox 360 controller 

### ISSUE THREAD

**Issue by cyian-1756** [ynsta/steamcontroller#45]
Is it possible to keep the right track pad in mouse mode (Drag to move) while emulating a Xbox controller?

**Comment by ynsta** [comment]
Yes my driver creates 3 devices a mouse a keyboard and a xbox pad. So you could mix the sc-xbox.py and sc-desktop.py scripts.

For example replace evm.setTrigAxis(Pos.RIGHT, Axes.ABS_RZ) by evm.setPadMouse(Pos.RIGHT).
and  then map the mouse button somewhere.

It depends on what functions of the xbox pad are not needed in your use case.
